x/wasm/client/cli/tx_test.go

```diff
@@ -57,3 +63,53 @@ func TestParseAccessConfigFlags(t *testing.T) {
 		})
 	}
 }
+
+func TestParseStoreCodeFlags(t *testing.T) {
+	anyValidWasmFile, err := os.CreateTemp(t.TempDir(), "wasm")
+	require.NoError(t, err)
+
+	anyValidWasmFile.Write(wasmIdent)
+
+	wasmBin, err := io.ReadAll(anyValidWasmFile)
+	require.NoError(t, err)
+
+	checkSum := sha256.Sum256(wasmBin)
```

Comment:
Please don't use the same code to generate the hash as it is used to verify it. Instead use the shell to hash it or some other source